### ISSUE TITLE
Generating revocations.efi documentation

### ISSUE THREAD

**Issue by 514amir** [rhboot/shim#662]
There is no explicit documentation I can find on how to generate revocations.efi which gets looked for by shim 15.8 at least during a PXE boot. It throws a warning and I'd prefer to not have the warning. I've seen the documentation on updating the revocation list but I am not sure how it applies to revocations.efi

**Comment by jsetje** [comment]
I have some docs on this coming, although the PXE warning is a pretty special case. If we go looking for the file during PXE, there will be a warning, since there is not readdir support there.

**Comment by TBP2k18** [comment]
It seems that combination of latest BIOSs for several Dell Precision series (3630, 3640) and shim 15.8 make revocations.efi mandatory (PXE boot sequence break immediately after revocations.efi load error). I found a temporary workaround for PXE boot, but I'm really interested in your documentation

**Comment by marta-lewandowska** [comment]
I have to agree with @christoph-at-unicon ... we are also getting questions and complaints about this from customers... I would be happy to help you with this @jsetje if you need/want. (:

**Comment by MrJohanssons** [comment]
What I don't get is why you don't get the message when booting Ubuntu 24.04 from USB? I mounted the ISO and couldn't find any **revocations.efi** file.

```
/mnt/EFI
└── boot
    ├── bootx64.efi
    ├── grubx64.efi
    └── mmx64.efi
```

**Comment by christoph-at-unicon** [comment]
I wonder if this is still an issue? It seems the initial request was implemented in 301cf52 ("Document how revocations can be delivered"). And my problem with the messages from a failed TFTP download was addressed in 2b49dc1 ("Suppress file open failures for some netboot cases").
